src/charm.py

```diff
@@ -132,24 +119,53 @@ def _update_fiveg_n3_relation_data(self, event: EventBase) -> None:
                 upf_ip_address=upf_access_ip_address,
             )
 
-    def _network_annotations_from_config(self) -> list[NetworkAnnotation]:
-        """Returns the list of network annotation to be added to the charm statefulset.
-
-        Annotations use configuration values provided in the Juju config.
-
-        Returns:
-            List: List of NetworkAnnotation objects.
-        """
+    def _network_attachment_definitions_from_config(self) -> list[NetworkAttachmentDefinition]:
```

Comment:
I know it's a private function, but to be aligned with the rest of the functions in the file it could be nice to add a docstring.

Reply:
Done